Already fixed in SVN

--- origsrc/attica-0.1.3/lib/cmake/libattica.pc.cmake	2010-04-06 11:55:37.000000000 -0500
+++ src/attica-0.1.3/lib/cmake/libattica.pc.cmake	2010-04-28 20:57:09.233071200 -0500
@@ -6,7 +6,7 @@ includedir=${CMAKE_INSTALL_PREFIX}/inclu
 Name: libattica
 Description: Qt library to access Open Collaboration Services
 #Requires:
-Version: ${ATTICA_VERSION_MAJOR}.${ATTICA_VERSION_MINOR}.${ATTICA_VERSION_PATCH}
+Version: ${CMAKE_LIBATTICA_VERSION_MAJOR}.${CMAKE_LIBATTICA_VERSION_MINOR}.${CMAKE_LIBATTICA_VERSION_PATCH}
 Libs: -L${LIB_DESTINATION} -lattica
 Cflags: -I${CMAKE_INSTALL_PREFIX}/include
 
